src/vulkan-renderer/wrapper/device.cpp

```diff
@@ -31,83 +33,334 @@ constexpr float DEFAULT_QUEUE_PRIORITY = 1.0f;
 } // namespace
 
 namespace inexor::vulkan_renderer::wrapper {
+namespace {
```

Comment:
Iam not a huge friend of those of unnamed namespaces. As i think some of the functions have a quite general behaviour and could be moved to `vk_tools/device_info.hpp` or similiar. In my eyes those functions are a bit unexpected here and a bit unorganized and lost overall.
Which already resulted into duplicated code :P (getting the device name).

Reply:
I am undecided about this. I will yeetari decide that :)

Reply:
Ok yeetari wants to keep it as it is, but I will move `get_physical_device_name` and `get_device_features_as_vector` to `vk_tools`.